### ISSUE TITLE
AVX2 support on VS2013 does not work

### ISSUE THREAD

**Issue by FrancescAlted** [Blosc/c-blosc#139]
Here it is the error that I am seeing using VS2013 Professional:

```
3>  shuffle-avx2.c
7>------ Rebuild All started: Project: test_getitem, Configuration: Release x64 ------
4>C:\Users\Francesc\Desktop\c-blosc2\blosc\shuffle-avx2.c(14): fatal error C1189: #error :  AVX2 is not supported by the target architecture/platform and/or this compiler.
2>C:\Users\Francesc\Desktop\c-blosc2\blosc\shuffle-avx2.c(14): fatal error C1189: #error :  AVX2 is not supported by the target architecture/platform and/or this compiler.
```

**Comment by FrancescAlted** [comment]
Of course, I am not an expert here, so perhaps @jack-pappas can shed some light on the validity of the fix.

**Comment by jack-pappas** [comment]
Why did you change the CMake check for the compiler version? VS2013 ships with Visual C++ (MSVC) 18.0.

I'm building blosc with no issues (and AVX2) using the existing CMake checks.

However, AVX2 support isn't available in VS2013 RTM - it was added in VS2013 Update 2. I will look for a way to detect the installed VS update version in CMake to handle this correctly in the case when someone is building with VS2013 RTM or Update 1.

**Comment by jack-pappas** [comment]
It looks like the compiler version for Vs2013 update 2 is 18.00.30501. That's the version we should check for in the CMake script.

**Comment by FrancescAlted** [comment]
Ah, that makes sense.  I am installing VS2013 update 4 now to check that and will correct that as soon as possible.  Thanks!

**Comment by FrancescAlted** [comment]
Addressed in 0d15680.
